src/nora/components/IconLink/IconLink.js

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+// TODO -- we should probably create a generic Link component :)
+const Link = ({ className, href, onClick, children, target = '_self' }) => {
+  const handleClick = (ev) => {
+    ev.preventDefault()
+    onClick(ev)
+  }
+  return (
+    <a href={href} className={className} onClick={handleClick} target={target}>
+      {children}
+    </a>
+  )
+}
+
+Link.propTypes = {
+  className: PropTypes.string,
+  href: PropTypes.string,
+  onClick: PropTypes.func,
+  children: PropTypes.node,
+  target: PropTypes.string,
+}
+
+export const IconLink = ({
+  iconPrefix,
+  iconName,
+  iconClassName,
+  iconContainerClassName,
+  linkClassName,
+  linkPosition,
+  copy,
+  onClick,
+}) => {
+  if (linkPosition === 'left') {
+    return (
+      <div className={iconContainerClassName}>
+        <Link href="#" className={linkClassName} onClick={onClick}>
```

Comment:
We'll want to make this more flexible on a later iteration, as we'll likely need it for the jump links part, but I'd rather implement it once I have this wired up to Nora so I can ensure it's doing what we want.